pymc3/model.py

```diff
@@ -657,6 +660,32 @@ def profile(self, outs, n=1000, point=None, profile=True, *args, **kwargs):
 
         return f.profile
 
+    def flatten(self, vars=None):
+        """Flattens model's input and returns:
+            * FlatView with input vector and replacements `input_var -> vars`
+            * ArrayOrdering
+            * view {variable: VarMap}
+
+        Parameters
+        ----------
+        vars : list of variables or None
+            if None, then all model.free_RVs are used for flattening input
+
+        Returns
+        -------
+        order, flat_view, view
```

Comment:
It seems conflicting with the code: `order, flat_view, view` -> `flat_view, order, view`